### DISCUSSION TITLE
Wondering about Double Tap

### DISCUSSION THREAD

**Discussion by gvest0b** [BishopFox/cloudfoxable#7] [General]
i am thinking about the double tap challenge - after getting keys for lambda_ec2 role, what can that entity do to obtain access to ec2_privileged ... lambda_ec2 primarily has ec2:* on resources where tag is double_tap1=true. but that also means creating resources is not possible, what other ec2 command there is that can change something or get a session. there is also not an existing ssh key i found to allow connecting on the ssh port. helpful ideas?

**Comment by sethsec-bf** [comment]
@girosvesting I assume you figured this out, right?

**Reply by gvest0b** [reply]
nah i did not really.  can't get to something that can put a key there.

**Reply by sethsec-bf** [reply]
Look into this permission: `ec2:ModifyInstanceAttribute`, see if there are any offensive ways to use that permission to your advantage :)

**Reply by gvest0b** [reply]
wowza! thanks you
slammed it
